### DISCUSSION TITLE
Actions not recognizing workflow file changes

### DISCUSSION THREAD

**Discussion by Jardo-51** [community/community#118709] [Actions]
### Select Topic Area

Bug

### Body

I added a workflow file into my `.github/workflows` repository.

In the "Actions" tab, I saw that the workflow file did not get parsed correctly. It said there was a "syntax error at line 10".

I fixed the error and pushed the changes, but the changes did not get recognized. The "Actions" tab still said "syntax error at line 10", but it was showing the old file content.

I checked the "Source" tab in Github, and the file had the correct content.

I tried editing the file directly in Github, but withe same result (Github still shows the old content).

I tried replacing the content of the workflow file with the content from a completely different workflow file (which is 100% correct) but withe same result (Github still shows the old content).

I tried deleting the file, but Github actions still shows it, with the old content.

I tried adding a completely different workflow file (different file name and content), but it is not shown in the "Actions" tab.

I think this is a bug in Github.

**Comment by ettaboyle** [comment]
@Jardo-51 you posted this discussion, and then it looks like you almost immediately closed it. Is this still an issue? If so, please re-open the discussion with a comment. If not, we are going to delete this post.
